src/App.js

```diff
@@ -1,5 +1,100 @@
+import { Console, Random } from "@woowacourse/mission-utils";
+
+const RandomBallNumber = () => {
```

Comment:
함수명은 camelCase로 작성하는게 더 좋을 것 같습니다!

https://github.com/airbnb/javascript#naming--camelCase


```suggestion
const randomBallNumber = () => {
```